### DISCUSSION TITLE
[Bug] Profile picture not showing

### DISCUSSION THREAD

**Discussion by josias-hosang** [swellstores/community#308] [Help] [answered]
Ahoi!

After changing my profile image in backend, just the alt-text is showing, because the resource URL set results in a 404 - not found: `https://cdn.swell.store/test-skipp/6459fa57f09ba900129bb868/99d4a2be80208b465409f26c0b549299?width=60&height=60`

**Comment by logeshswell** [comment]
@josias-hosang Looks like an upload error when you change your profile picture. Can you try uploading it once again?

**Reply by josias-hosang** [reply]
I already tried several different images also of different sizes. Looks always the same.

**Reply by logeshswell** [reply]
I will file a bug ticket to fix this at the earliest.

**Reply by logeshswell** [reply, marked as answer]
This issue is resolved now. Can you check and confirm?
